comet/broker.py

```diff
@@ -14,6 +14,7 @@
 """
 import aioredis
 import asyncio
+import contextvars
```

Comment:
Is this import necessary? It doesn't seem to be used.

Reply:
contextvars? It's used in line 59 to keep request IDs.

Reply:
My mistake. I didn't spot where it was moved from.